packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -233,7 +233,9 @@ const handleEnterKeyOnList = (editor) => {
 };
 
 const Img = styled.img`
+  max-height: 512px;
```

Comment:
this number looks arbitrary – i'm sure it's not, can you write a comment where this number comes from e.g. design